src/pages/index.js

```diff
@@ -0,0 +1,361 @@
+import {
+  enableValidation,
+  validationConfig,
+  resetValidation,
+  disableBtn,
+} from "../scripts/validation.js";
+import "../pages/index.css";
+import Api from "../utils/api.js";
+import { setButtonText } from "../utils/helper.js";
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "0461a93f-e4a4-4710-9671-530f1a6885c6",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([userInfo, cards]) => {
+    //Handle the user's information
+    // - set the src of the avatar image
+    document.querySelector(".profile__avatar").src = userInfo.avatar;
+    // - set the text content of both the name and description
+    document.querySelector(".profile__name").textContent = userInfo.name;
+    document.querySelector(".profile__description").textContent =
```

Comment:
`document.querySelector(".profile__description")` 

This has already been found as `descriptionElement` in line 55. Please, reuse it here


```js
    nameElement.textContent = userInfo.name;
    descriptionElement.textContent = userInfo.about;
```